changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version Next
 -------------
+- [PATCH] Adding Credential Manager version (for compilation only) (#1772)
```

Comment:
`for compilation only`

So it doesn't ship? IIUC, if common is using this then this should still make it into the final APK?

Reply:
It does ship, but I didn't want it to seem that ADAL is actually getting this feature (ADAL uses a different set of classes for WebView). I'll change this changelog to cut the compilation part and just mention that ADAL isn't getting the feature.